docs/dev/cli/saucectl/usage/ci/github-actions.md

```diff
@@ -12,8 +12,9 @@ These examples can apply to virtually any GitHub deployment, provided that you a
 
 ## What You'll Need
 
-* [GitHub Account](https://github.com/join)
-* [Sauce Labs Account](https://saucelabs.com/sign-up)
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
```

Comment:
Should all list item end with a period?

Reply:
Same comment as https://github.com/saucelabs/sauce-docs/pull/1100#discussion_r825599689